packages/discord-request/src/client.ts

```diff
@@ -1,14 +1,22 @@
-import pkg from "../package.json" assert { type: "json" };
-import type { Prettify, RequestData, RequestOptions } from "./definitions.js";
-import { AUDIT_LOG_LIMIT, RequestMethod } from "./definitions.js";
+import type {
+	Prettify,
+	RequestData,
+	RequestOptions,
+	UserAgent,
+} from "./definitions.js";
+import {
+	AUDIT_LOG_LIMIT,
+	DEFAULT_USER_AGENT,
+	RequestMethod,
+} from "./definitions.js";
 import { request } from "./request.js";
 
 export type ClientConfig = {
 	api: string;
 	cdn: string;
 	headers: Record<string, string>;
 	timeout: number;
-	userAgent: `DiscordBot (${string}, ${string})${string}`;
+	userAgent: string;
```

Comment:
why string instead of UserAgent?

Reply:
This is what the user is passing in, which is appended to the one we set now - since it's excess info, it can just be a string format